SpatialGDK/SpatialGDK.uplugin

```diff
@@ -0,0 +1,35 @@
+{
+	"FileVersion": 3,
+	"Version": 1,
+	"VersionName": "1.0",
+	"FriendlyName": "SpatialGDK",
+	"Description": "SpatialOS Unreal GDK",
+	"Category": "SpatialOS Unreal GDK",
+	"CreatedBy": "Improbable Worlds, Ltd.",
+	"CreatedByURL": "https://improbable.io",
+	"DocsURL": "",
+	"MarketplaceURL": "",
+	"SupportURL": "https://forums.improbable.io/c/sup",
+    "EnabledByDefault" : true,
+	"CanContainContent": true,
+	"IsBetaVersion": false,
+	"Installed": true,
+	"Modules": [
+		{
+			"Name": "SpatialGDK",
+			"Type": "Runtime",
+			"LoadingPhase": "Default",
+            "WhitelistPlatforms": [ "Win64", "Linux" ]
```

Comment:
Add "XboxOne" and "PS4"?

Reply:
I haven't been able to test those but I guess we could add them